.github/workflows/kakarot_release.yml

```diff
@@ -121,17 +121,26 @@ jobs:
             --push
 
   build-push-indexer-docker:
-    runs-on: ubuntu-latest-16-cores
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
     strategy:
       matrix:
         include:
           - platform: linux/amd64
-            apibara_sink_bin_dir: sci7y4a8r7yz05094pl88h3d7vv1s98f
+            apibara_download_url: https://github.com/apibara/dna/releases/download/sink-mongo%2Fv0.7.0/sink-mongo-x86_64-linux.gz
```

Comment:
Shouldn't we modify this as well in the hive docker file and in the README? I remember that we have something related in there.

Reply:
ah sorry Clement merged, it looks like the only place we still use it is hive, which for now still works, let's keep it for now, we can update it if hive starts failing